cmd/access/node_builder/access_node_builder.go

```diff
@@ -492,13 +497,19 @@ type Option func(*AccessNodeConfig)
 
 func WithBootStrapPeers(bootstrapNodes ...*flow.Identity) Option {
 	return func(config *AccessNodeConfig) {
-		config.bootstrapIdentites = bootstrapNodes
+		config.bootstrapIdentities = bootstrapNodes
 	}
 }
 
-func WithUnstakedNetworkBindAddr(bindAddr string) Option {
+func SupportsUnstakedNode(enable bool) Option {
 	return func(config *AccessNodeConfig) {
-		config.unstakedNetworkBindAddr = bindAddr
+		config.supportsUnstakedFollower = enable
+	}
+}
+
+func WithNetworkKey(key crypto.PrivateKey) Option {
```

Comment:
for the consensus follower, the network key will be passed in as a param instead of being read from a bootstrap file.